panel/chat/langchain.py

```diff
@@ -162,3 +162,16 @@ def on_retriever_end(self, documents, **kwargs: Any) -> Any:
     def on_text(self, text: str, **kwargs: Any):
         """Run when text is received."""
         return super().on_text(text, **kwargs)
+
+    def on_chat_model_start(
```

Comment:
What is the purpose of adding this method? As far as I can see it does not do anything?

Reply:
See the docstring.

Reply:
Updated the docstring to be clearer.